tests/unit/bin/init.bats

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bats
```

Comment:
Sorry if I've already asked this before. Why do we need this test when we have the same tests as template tests in `init.bats.yaml`?

Reply:
It is one one of the tests that are overlapping, and that will be changed when the `none` cloud provider is added as I will change this one to use it instead.

But the reason why it still was included was because it makes more sense to have the test first that all files are created, then in a second test check that all files are backed up.

Reply:
Makes sense!